modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,402 @@
+
+* Format and mount each dedicated disk.
+* Add mount entries for each disk to `/etc/fstab`.
+* Add iptable rules.
+* Enable required kernel modules.
+* Start the installation.
+
+The controller VM that is the leader for installation performs the following actions:
+
+* Runs the activation script after installation.
+* Runs the Mule license insertion script after registration.
+
+== Anypoint Platform Roles and Permissions
+To succesfully use Anypoint Runtime Fabric, your Anypoint Platform account must have the following roles and permissions enabled:
+
+* Ability to use Anypoint Runtime Manager to deploy and manage applications
+* Ability to use Anypoint Access Management to manage permissions for Anypoint Platform users
+** To use Anypoint Runtime Fabric, you must have the *Organization Administrators* role or the *Manage Runtime Fabrics* permission on the corresponding environments.
+* You might also need the following permissions to enable inbound traffic to Runtime Fabric:
+
+** Grant access to secrets
+** Manage Secret Groups
+** Read secrets metadata
+** Write secrets
++
+These permissions are available from the *Secrets Manager* tab in Anypoint Access Management.
+** To deploy applications, you must have the *Exchange Contributors* role enabled for your Anypoint Platform account.
+
+== Production Configuration Requirements
+
+A minimum of 6 nodes are required for a production configuration of Runtime Fabric, as shown in
+the following diagram.
+
+image::architecture-production.png[]
+
+The following sections describe the minimum system requirements of controller and worker
+nodes in a production configuration.
+
+[NOTE]
+Runtime Fabric does not support running controller nodes using burstable VMs. When deploying an application on a worker node, you can allocate vCPU for bursting. See xref:deploy-resource-allocation.adoc[Resource Allocation and Performance on Anypoint Runtime Fabric] for additional information.
+
+=== Controller Nodes
+
+The minimum number of controller nodes required for a production environment is three. This requirement reflects 
+the minimum requirements to maintain high-availability and ensure system performance.
+
+An odd number of controller nodes is required for fault tolerance. Even numbers of controller nodes are not supported. 
+
+The maximum number of supported controller nodes is 5.
+
+Each controller node must have the following resources:
+
+** A minimum of 2 dedicated cores.
+** A minimum of 8 GiB memory for each dedicated core.
++
+You must consider the amount of resources needed for the internal load balancer when sizing controller nodes. For additional information, refer to xref:deploy-resource-allocation#internal-load-balancer[Internal Load Balancer] resource allocation.
+** A minimum of 80 GiB dedicated disk for the operating system.
```

Comment:
Fix this and all subsequent bullet levels. You cannot use ** if you do not use * first.

Reply:
GLOBALLY remove periods from the following list items.